EVM and High Level Languages.md

```diff
@@ -0,0 +1,95 @@
+
+Ethereum Blockchain technology offers limitless use case scenarios. A virtual computer deployed on the Ethereum Blockchain creates many possible use cases. Smart contracts are programs written in a high-level programming language. Think of JavaScript syntax with a name of Solidity or Vyper.  These languages build Smart contract code that compiles for use on the Ethereum Blockchain. This computing system revolutionizes privacy, security, and freedom otherwise unrealized in present-day governance.
+
+## Ethereum Virtual Machine Feature Overview
+
+High-level is a vague description of a computing platform's capability. Precise performance metrics would go beyond the scope of this article. The Ethereum Blockchain is a powerful and isolated virtual computing environment. Computing features of the Ethereum Virtual Machine (EVM) include: 
+
+* Predefined function libraries 
+* Function constructors and destructors
+* Function overrides 
+* High-level logic
+* Looping and recursion 
+* Full range of expressions for logical and mathematical arguments
+* Control structures
+* More than one variable can pass in and out of a function 
+* User ability to define custom functions 
+* Inheritance
+* Polymorphism 
+* Sand boxing 
+* Two-way interaction with other smart contracts 
+
+Simple transaction protocol initiates the deployment of very powerful applications on the EVM. The Ethereum protocol is trust-less; secured by peer-to-peer consensus and data encryption. It is very difficult, impossible, to access unauthorized information. The program code is verifiable and impossible to alter. Always available to every user as the same version and build. Pushing updates that users ignore can become a thing of the past.
+
+Prebuilt functions are built in the EVM.  Among other things these functions create data abstraction between the user and the actual blockchain implementation.  Direct features of the EVM are listed below that have an aggregate functionality of a self contained computer system.
+
+* Simple transaction methods for new smart contract deployment
+* Smart contract interaction only by address
+* Arguments and data pass in from the user and pass out to a user or contract
+* A stack type data structure holds local variables
+* It reads function call arguments and return-addresses
+* Logic functions can change the order in which instructions execute
+* Addressable persistent storage acts as a virtual hard drive 
+* Non-persistent storage acts as system memory
+
+A closer look at the EVM and dApp ecosystem show a complex orchestration of silently running programs and processes. Technology today offers many choices in a programming language presenting obstacles in compatibility. Cross-platform data abstraction requires a complex web of behind the scenes computing. 
```

Comment:
This paragraph isn't really helpful. Maybe it's better to reword to explain that contracts interact with each other build from each other.